torch/csrc/autograd/VariableTypeUtilsDependOnOps.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <ATen/core/boxing/KernelFunction.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <torch/csrc/jit_decomp_interface.h>
+
+// This is the set of helpers in VariableTypeUtils have a dependency on
+// native_functions.yaml meaning the file will need to be re-compiled every time
+// an operator is changed or added. We cannot simply put these functions in
+// VariableType.h and VariableTypeutils.h, since they are included in files like
+// ADInplaceOrViewType_X.cpp which don't always want to be recompiled.
```

Comment:
This message might be out of date now, we could probably merge this file back into VariableTypeUtils.h

Reply:
This file used to depend on decomposition_registry directly which depends on more jit things which depend on interned strings.

torch/csrc/jit/ir/ir.h -> aten/src/ATen/core/interned_strings.h